buildchain/buildchain/constants.py

```diff
@@ -126,5 +126,14 @@ def git_ref() -> Optional[str]:
 )
 GO_SOURCES: FrozenSet[Path] = frozenset(filepath for filepath in ROOT.rglob("*.go"))
 
+OPERATOR_SDK_GENERATE_CMDS: List[List[str]] = [
+    [
+        config.ExtCommand.OPERATOR_SDK.value,
+        "generate",
+        "k8s",
+    ],
```

Comment:
nit(style): why is this command split into multiple lines, but not the following one?

Reply:
For .... no reason :smile:

Reply:
Fixed